Samples/PublicSamples/RecordingBot/docs/tutorials/deploy/aks.md

```diff
@@ -0,0 +1,384 @@
+# Deploy an AKS cluster
+
+We'll start with deploying an AKS cluster, on the AKS cluster we later deploy the containers with the sample recording bot. Before we can start to run commands in the Azure command line tool, we have to login, to do so, we run:

```

Comment:
```suggestion
Start by deploying an AKS cluster. Before you can start to run commands in the Azure CLI tool, you have to login. To do so, run:

```

Reply:
Diátaxis suggests to use the "we" form, the learner does not go alone threw the tutorial we the teacher and the learner together do a tutorial